wagtailio/project_styleguide/templates/patterns/components/related-content/related-content.html

```diff
@@ -0,0 +1,13 @@
+{% if related_pages %}
+    <div class="related-content grid">
+        <h3 class="related-content__heading">You might also like</h3>
```

Comment:
Not sure if this is hardcoded or not - couldn't find a corresponding field for it.